content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -0,0 +1,87 @@
+# Persisting Experiments
+
+DVC runs experiments outside of the standard Git stage/commit workflow for quick
+iteration. In this section we'll see how to bring the experiments we run to the
+standard Git workflow with `dvc exp branch` and `dvc exp apply`.
```

Comment:
Good, thanks! Just one thing: It's a bit repetitive to use the "std git flow" phrase in both sentences. Do we even need to focus so much on Git? Maybe there's a more general way to state this in the first sentence (then it can be OK to be specific about git flow in the 2nd one).